gr-vocoder/lib/CMakeLists.txt

```diff
@@ -104,6 +104,9 @@ endif(LIBGSM_LIBRARIES)
 
 add_library(gnuradio-vocoder SHARED ${gr_vocoder_sources})
 target_link_libraries(gnuradio-vocoder ${vocoder_libs})
+if(LIBGSM_INCLUDE_DIR)
+    target_include_directories(gnuradio-vocoder PUBLIC ${LIBGSM_INCLUDE_DIR})
```

Comment:
@noc0lour that's not modern CMake, is it? How do I fix it?

Reply:
the FindGSM might need some touch ups (: